src/main/java/com/esri/samples/mapview/scale_bar/ScaleBarSample.java

```diff
@@ -0,0 +1,80 @@
+package com.esri.samples.mapview.scale_bar;
+
+import com.esri.arcgisruntime.UnitSystem;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.toolkit.Scalebar;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+
+public class ScaleBarSample extends Application {
+
+    private MapView mapView;
+
+    private static final double LATITUDE = 64.1405;
```

Comment:
Since these are only used once you can put the values directly in the map constructor and remove the static fields. We need to go back and do this for our other samples